app/javascript/packages/components/troubleshooting-options.spec.tsx

```diff
@@ -81,4 +81,12 @@ describe('TroubleshootingOptions', () => {
     const tag = getByText('components.troubleshooting_options.new_feature');
     expect(tag.classList.contains('text-uppercase')).to.eq(true);
   });
+
+  it('renders with no divider class when overridden', () => {
+    const { baseElement } = render(
+      <TroubleshootingOptions {...DEFAULT_PROPS} isNewFeatures divider={false} />,
+    );
+
+    expect(baseElement.querySelector('.full-screen.bg-white')).to.not.exist();
```

Comment:
> Why assert `troubleshooting-options` presence, too?

For me, a combination of:

1. Making sure we're adding the class, since it's not already covered by another spec, and it's necessary for it to be added for the element to be styled correctly
2. As a confidence check that we're asserting against the expected element, since at least in the second test case `expect(element.classList.contains('troubleshooting-options--divider')).to.be.false()` could pass if `element` were something completely different than what we expect (e.g. `document.body`). The fact we have to dig through the DOM to get to the element with `firstElementChild` makes it more of a concern for me.